src/display/editor/signature.js

```diff
@@ -244,27 +244,18 @@ class SignatureEditor extends DrawingEditor {
   }
 
   /** @inheritdoc */
-  async addEditToolbar() {
-    const toolbar = await super.addEditToolbar();
-    if (!toolbar) {
-      return null;
-    }
-    if (this._uiManager.signatureManager && this.#description !== null) {
-      await toolbar.addEditSignatureButton(
-        this._uiManager.signatureManager,
-        this.#signatureUUID,
```

Comment:
Interesting; AFAICT `addEditSignatureButton` only takes one parameter, so I assume the other two here have never actually been used?

Reply:
Yep it was useless.